docs/integrations/databases/hasura.mdx

```diff
@@ -1,5 +1,166 @@
 ---
-sanity_slug: /docs/integration/hasura
+title: Hasura
+description: Learn how to integrate Clerk into your Hasura application.
 ---
 
-# This is a stub
+# Getting started
+
+The first step is to create a new Clerk application from your Clerk Dashboard if you haven’t done so already. You can choose whichever authentication strategy and social login providers you prefer. For more information, check out our [Set up your application](/docs/quickstarts/setup-clerk) guide.
+
+After your Clerk application has been created, navigate to the [JWT Templates](https://dashboard.clerk.com/last-active?path=jwt-templates) screen from the Clerk Dashboard. Click on the **New template** button to create a new template based on Hasura.
+
+<Images 
+  width={3024}
```

Comment:
These images are too big and could be scaled down. I think the max size of container that the images are put in are less than 1000px (I think the breakpoint where the sidebar disappears is where the container is largest).

Reply:
Sure, these are just the natural width and height on the image properties. I can cut them in half - but I'd appreciate to be educated on the why for this suggestion!

Reply:
It's largely an optimization issue; we are loading images that are much larger than needed.  This leads to an increased file size that takes longer to load,  but we do not benefit from the increased image size because the browser has to scale the larger image down to fit in its container.

![CleanShot 2023-09-22 at 10 48 02@2x](https://github.com/clerkinc/clerk-docs/assets/2105067/2142c85f-4de4-4728-a731-1501f6df903c)

Reply:
Oh yes this part makes sense, but I thought Next/Image handled this 😭 there are so many images that need to be fixed 😭😭😭

Reply:
but thank you, this makes a lot of sense I should have thought about this I learned this along time ago!! totally slipped my mind